istioctl/cmd/add-to-mesh.go

```diff
@@ -119,8 +119,9 @@ func deploymentMeshifyCmd() *cobra.Command {
 	var opts clioptions.ControlPlaneOptions
 
 	cmd := &cobra.Command{
-		Use:   "deployment <deployment>",
-		Short: "Add deployment to Istio service mesh",
+		Use:     "deployment <deploy>",
+		Aliases: []string{"deploy"},
```

Comment:
Did you think about "dep" or "d"?

Reply:
`deploy` is the alias used in `kubectl`. I think it's intuitive if we use the same alias.

Reply:
As @esnible suggested, I think we can add both the aliases `[]string{"deploy", "d"}`. In this way, it will be aligned with `kubectl` + more options?

Reply:
@shamsher31 I've added "dep". Thanks